documentation/src/docs/controller-service.md

```diff
@@ -74,15 +75,15 @@ more than one instance of Controller service per cluster.
 
 Each Controller instance is capable of working independently and uses a
 shared persistent store as the source of truth for all state-owned and
-managed by Controller service. We currently use Apache ZooKeeper as the
-store for persisting all metadata consistently.  Each instance comprises
+managed by Controller service. We use [Pravega KeyValue Tables](https://github.com/pravega/pravega/wiki/PDP-39-Key-Value-Tables) as the
+store for persisting all metadata consistently. Controller stores metadata about streams in stream specific tables that creates. Each controller instance comprises
```

Comment:
typo: ```Controller stores metadata about streams in stream specific tables that **it** creates```

Reply:
```Each controller instance comprises various subsystems which are responsible for performing specific operations on different categories of metadata. ```
I think removing the "which are" would make this statement more easily readable.
Alternatively it may be good to have a separate subsection on Controller Subsystems and list them there explaining in 1 line the function of each subsystem.